### PR TITLE
Fix #579 (formatting melk)

### LINKED ISSUE #579: Reformat all *.melk files with the formatter
To have a single-point of "history-destruction", we should reformat all *.melk files using the implemented formatter. So far this has only been done for the Core.melk.

### PR DESCRIPTION
**Pull request by uruuru**
I slightly altered the formatter (without changing the out-of-conventions coding style of the class):
  - no space between layout algorithm id and its implementing class
  - two newlines between groups, options, etc.
  - indent text to 'description' of categories, options, and algorithms
  - no space before commas

Afterwards I reformatted all of the `*.melk` files. One major change is that tabs are replaced by spaces (although that's an editor configuration). 

Anything that feels awkward?

Note that the changes of the formatter are not available within the main eclipse until a new version has been deployed and installed.

### CODE REVIEW

File: plugins/org.eclipse.elk.alg.force/src/org/eclipse/elk/alg/force/Force.melk [hunk 1]
```diff
@@ -33,8 +33,7 @@ algorithm force(ForceLayoutProvider) {
     features multi_edges, edge_labels
     preview images/force_layout.png
     supports org.eclipse.elk.priority = 1
-	   documentation 
-	       "Priorities set on nodes determine the order in which connected components are placed:
+    documentation "Priorities set on nodes determine the order in which connected components are placed:
```
Comment (le-cds) [inline plugins/org.eclipse.elk.alg.force/src/org/eclipse/elk/alg/force/Force.melk:36]: Why is the newline removed here, but added in `Disco.melk`?

Comment (uruuru) [reply plugins/org.eclipse.elk.alg.force/src/org/eclipse/elk/alg/force/Force.melk:36]: Are you maybe confusing `documentation` and `description`? 

`documentation` should (usually) refer to some markdown file, e.g. `@docs/myOption.md`. As such, I thought one single line should be fine. 
However, there are some rare cases where text is provided directly.

Comment (le-cds) [reply plugins/org.eclipse.elk.alg.force/src/org/eclipse/elk/alg/force/Force.melk:36]: Oh... so I am. Never mind me, then. 🤦

**Comment by uruuru** [comment]
Good to merge?

**Pull request merged**